examples/gin/README.md

```diff
@@ -2,10 +2,26 @@
 
 This is an example of basic Gin app with Airbrake middleware that reports route stats.
 
-## How to run API
+## How to run Exampe API
+
+Copy the example in your testing folder.
+
+Insert youy project ID and project key in main.go file
+
+run:
 
 ```bash
-go run *.go -env=production -airbrake_project_id=123456 -airbrake_project_key=FIXME
+go mod init {directory-name}
```

Comment:
I think we could provide a test name so that users don't have to think hard about it.